README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/gap-packages/io.svg?branch=master)](https://travis-ci.com/gap-packages/io)
+![CI](https://github.com/ssiccha/io/workflows/CI/badge.svg)
```

Comment:
That needs to be adjusted. Also there shouldn't be just a badge, it should also link to the CI results for this workflow on the master branch (as I did for e.g. gap-system/gap).

Reply:
Ping? @ssiccha any plans to update this PR?

Reply:
PR is updated now. The badge now is:
```
[![CI](https://github.com/ssiccha/io/workflows/CI/badge.svg)](https://github.com/gap-packages/recog/actions?query=workflow%3ACI+branch%3Amaster)
```